internal/jobqueue/job.go

```diff
@@ -21,28 +22,28 @@ type JobStatus struct {
 	Status string `json:"status"`
 }
 
-func (job *Job) Run() error {
+func (job *Job) Run() (error, []error) {
```

Comment:
What about []error as a single return type? Returning `nil` if no error happened, `[]error{err}` when one error happened, `r` in other cases.

Reply:
I like having the array matching the targets array, so that eventually it becomes eaiser to return per-target status info to osbuild-composer and match it to the individual targets. Would be fine to ignore the first one, but I prefer to make it explicit.